test/unit/spec/preflight.ts

```diff
@@ -1,9 +1,9 @@
 import * as assert from 'assert';
-import { Preflight } from '../../../lib/preflight';
+import { runPreflight } from '../../../lib/preflight/preflighttest';
 
 describe('Preflight', () => {
   it('should return a Preflight object', () => {
-    const preflight = new Preflight();
+    const preflight = runPreflight('foo', {});
     assert(!!preflight);
   });
 });
```

Comment:
Do we have plans to add more unit tests?

Reply:
no because we have very good coverage with integration tests.

Reply:
Now I have added more tests for new ts files.